src/ec/packages/ec-component-text-area/text-area.html.twig

```diff
@@ -10,6 +10,7 @@
   - "hide_label" (boolean) (default: '')
   - "placeholder" (string) (default: '')
```

Comment:
I've checked and i don't see any placeholder in ECL textarea, but here it comes a question: we had tickets for updating form fields, did we touch also the text-area or was it forgotten..?

Reply:
done, checking to update other components too..